src/python/grpcio/grpc/experimental/aio/_server.py

```diff
@@ -83,35 +86,32 @@ def add_secure_port(self, address: Text,
         """
         await self._server.start()
 
-    def stop(self, grace: Optional[float]) -> asyncio.Event:
+    async def stop(self, grace: Optional[float]) -> None:
         """Stops this Server.
 
-        "This method immediately stops the server from servicing new RPCs in
+        This method immediately stops the server from servicing new RPCs in
         all cases.
 
-        If a grace period is specified, this method returns immediately
-        and all RPCs active at the end of the grace period are aborted.
-        If a grace period is not specified (by passing None for `grace`),
-        all existing RPCs are aborted immediately and this method
-        blocks until the last RPC handler terminates.
+        If a grace period is specified, all RPCs active at the end of the grace
+        period are aborted.
 
-        This method is idempotent and may be called at any time.
-        Passing a smaller grace value in a subsequent call will have
-        the effect of stopping the Server sooner (passing None will
-        have the effect of stopping the server immediately). Passing
-        a larger grace value in a subsequent call *will not* have the
-        effect of stopping the server later (i.e. the most restrictive
-        grace value is used).
+        If a grace period is not specified (by passing None for `grace`), all
```

Comment:
Rationale on the change in behavior with respect to repeated calls with a specified grace period?